int16[] cluster
int16[] tlapses
float32[] decay_rates
---
bool availability